apps/hpc-cdm/src/app/components/form-assignments-list.tsx

```diff
@@ -96,9 +96,9 @@ const FormAssignmentsList = (props: Props) => {
                         title={`${dayjs(a.lastUpdatedAt)
                           .locale(lang)
                           .format('ddd D MMM YYYY LTS')}`}
-                          classes={{
-                            tooltip: classes.tooltip,
-                          }}
+                        classes={{
+                          tooltip: classes.tooltip,
+                        }}
```

Comment:
In 5d4eef5583ac76359279e582fad41a74323b0ee1, you have changes to two `webpack.js` files which are the only ones belonging there. Rest of the changes are fixes of mistakes from previous commits. Don't commit those separately, but go back, rebase and amend (change) the relevant commits where changes like this formatting belong.

As a useful rule of thumb, any time you write "and" in your commit message, you are likely doing something that doesn't belong in that commit :)